routes/login/login.spec.js

```diff
@@ -41,19 +41,20 @@ describe('Test /login responses', () => {
         .post('/login/code')
         .send({ redirect: '/' })
       const $ = cheerio.load(response.text)
+      expect($('title').text()).toMatch(/^Error:/)
       expect($('.error-list__header').text()).toEqual('Please correct the errors on the page')
       expect($('.error-list__list').children()).toHaveLength(1)
-      expect($('.validation-message').text()).toEqual('Access code must be 8 characters')
-      expect($('#code').attr('aria-describedby')).toEqual('code_error')
+      expect($('.validation-message').text()).toEqual('Error: Access code must be 8 characters')
+      expect($('#code').attr('aria-describedby')).toEqual('code--error')
```

Comment:
what about having this class just be `code-error`? `--` is BEM thing which assumes this is a variant of code

Reply:
Sure, that works.